src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,30 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
+	
+	private static final String PRINT_FORMAT = "%s %s %s";
+	private static final String ADDED_SIGN = "+";
+	private static final String DELETED_SIGN = "-";
```

Comment:
In practice, it helps if you put the common part of the name first. That way, related names appear together when sorted alphabetically. e.g. `SIGN_ADDED`